CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 1.11.0-beta-1
+- ABI Interaction Support for JAVA SDK (#268)
+- Bug fix for `logs` on `PendingTransactionResponse` (#275)
+- Add WaitForConfirmation function (#274)
+- Better error message on encoding exception. (#258)
+- Revert "Revert "Fix ABI source code position for ABI feature (#260)""
+- Revert "Revert "Add ABI encoding support (#255)""
```

Comment:
nit: could you remove the revert language from these? E.g. `Fix ABI source code position for ABI feature (#260)` is a good description (we reverted & un-reverted some commits around the previous release)